src/model/block/ISetContactPropertyBlockConfig.ts

```diff
@@ -29,13 +29,13 @@ export interface SetContactProperty {
 export interface ISetContactPropertyBlockConfig extends IBlockConfig {}
 
 export interface ISetContactPropertyBlockConfigRequired extends ISetContactPropertyBlockConfig {
-  set_contact_property: SetContactProperty
+  set_contact_property: SetContactProperty[]
```

Comment:
Shall we consider renaming the attribute to `set_contact_properties` to indicate this should be an array? This will also result in cleaner full-text search over codebase 🙂

Reply:
@object-required, that was the 1st idea. But [Mark proposed to keep it](https://votomobile.slack.com/archives/CMQDVRDN3/p1656948302778729?thread_ts=1656948302.778729&cid=CMQDVRDN3) by saying "keep the name of these field" which we found reasonable as well.